app/grandchallenge/emails/admin.py

```diff
@@ -0,0 +1,98 @@
+from django.contrib import admin, messages
+from django.contrib.auth import get_user_model
+from django.utils.timezone import now
+from django.utils.translation import ngettext
+from markdownx.admin import MarkdownxModelAdmin
+
+from grandchallenge.algorithms.models import Algorithm
+from grandchallenge.challenges.models import Challenge
+from grandchallenge.emails.models import Email
+from grandchallenge.emails.tasks import send_bulk_email
+from grandchallenge.reader_studies.models import ReaderStudy
+
+
+def schedule_emails(modeladmin, queryset, request, receivers, action_string):
+    emails = queryset.filter(sent=False)
+    if emails:
+        for email in emails:
+            send_bulk_email(receivers, email.subject, email.body)

```

Comment:
This isn't actually running the task through celery, you're calling the task synchronously here rather than using `apply_async`.